aws-java-workshop/src/main/java/com/emc/vipr/s3/sample/_02_ReadObject.java

```diff
@@ -15,6 +15,8 @@
 package com.emc.vipr.s3.sample;
 
 import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.GetObjectMetadataRequest;
```

Comment:
IJ automatically adds imports...There is some code that might be commented out at this time, but is frequently uncommented and used during the workshop and that can result these.